k9mail-library/build.gradle

```diff
@@ -11,6 +11,7 @@ if (rootProject.testCoverage) {
 }
 
 dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:${kotlinVersion}"
```

Comment:
extra commit

Reply:
I wanted to use Kotlin in the `k9mail-library` module. I didn't feel like that change warranted a separate commit.